api.planx.uk/modules/admin/session/csv.test.ts

```diff
@@ -12,13 +13,18 @@ const mockGenerateCSVData = vi.fn().mockResolvedValue([
     metadata: {},
   },
 ]);
-vi.mock("@opensystemslab/planx-core", () => {
+
+vi.mock("@opensystemslab/planx-core", async (importOriginal) => {
```

Comment:
This pattern is repeated a few times here, grouped in b1d6037b5cabd11ae672d29fd88790625807f3e4

Now that JWT verification needs to make a round trip to the DB, this test needs to access a GraphQL client to do this (`$api.client.request`) which is not mocked. Using `importOriginal()` we can only mock the function we want to mock in this test suite and keep the rest of the original behaviour. In this case it's `this.export.csvData`.

We have an open issue to consolidate these mocks and to make them easier and simpler to manage - the pattern here could be DRY'ed up a but into a `setupPlanXCoreMock()` function called by vitest setup. I've not tackled this here in order to make process and keep things separate. The issue is here - https://github.com/theopensystemslab/planx-new/issues/2404